__tests__/validation.test.js

```diff
@@ -0,0 +1,59 @@
+import {
+  validateCarNames,
+  validateRoundCount,
+} from '../src/services/validation.js';
+import { ERROR_MESSAGES } from '../src/constant.js';
+
+describe('사용자 입력 검증', () => {
+  describe('자동차 이름 입력 검증', () => {
+    test('자동차 이름 입력은 공백일 수 없다', () => {
+      expect(() => validateCarNames('')).toThrow(ERROR_MESSAGES.emptyCarName);
+    });
+
+    test('쉼표(,) 이외의 유효하지 않은 구분자는 자동차 이름 입력에 포함될 수 없다', () => {
+      expect(() => validateCarNames('car1,car2;car3')).toThrow(
+        ERROR_MESSAGES.invalidDelimiter
+      );
+    });
+
+    test('쉼표(,) 이외의 특수문자, 공백을 자동차 이름 입력에 포함할 수 없다', () => {
+      expect(() => validateCarNames('car@1,car2')).toThrow(
+        ERROR_MESSAGES.invalidDelimiter
+      );
+    });
```

Comment:
`test.each`를 활용해서 여러 테스트 케이스를 간단하게 실행해보는 것도 좋을 것 같아요
```suggestion
    test.each([
      {
        carNames: '',
        errorMessage: ERROR_MESSAGES.emptyCarName
      },
      {
        carNames: 'car1,car2;car3',
        errorMessage: ERROR_MESSAGES.invalidDelimiter,
      },
      {
        carNames: 'car@1,car2',
        errorMessage: ERROR_MESSAGES.emptyCarName,
      },
    ])('%s에 대한 테스트', ({ carNames, errorMesage }) => {
      expect(() => validateCarNames('')).toThrow(ERROR_MESSAGES.emptyCarName);
    });